packages/eas-cli/src/commands/submit.ts

```diff
@@ -111,8 +148,11 @@ export default class BuildSubmit extends Command {
 
         // ios
         'apple-id': appleId,
-        'apple-app-specific-password': appleAppSpecificPassword,
-        'app-apple-id': appAppleId,
+        'asc-app-id': appAppleId,
```

Comment:
```suggestion
        'asc-app-id': ascAppId,
```
And maybe change all other `appAppleId` occurrences with `ascAppId` so we don't use a few names for the same thing.

Reply:
One place I have to leave `appAppleId` is the API request payload